assets/src/domain/eventEditor/ui/datetimes/datesList/cardView/Details.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useCallback } from 'react';
+import { __ } from '@wordpress/i18n';
+
+import { InlineEditHeading, InlineEditTextArea } from '@appInputs/InlineEditInput';
+import DateDetailsPanel from './DateDetailsPanel';
+import { useDatetimeMutator } from '@edtrServices/apollo/mutations';
+import type { Datetime } from '@edtrServices/apollo/types';
+import { getPropsAreEqual } from '@appServices/utilities';
+
+interface DetailsProps {
+	datetime: Datetime;
+}
+
+const Details: React.FC<DetailsProps> = ({ datetime }) => {
+	const { updateEntity } = useDatetimeMutator(datetime.id);
+
+	const onChangeName = useCallback(
+		(name: string): void => {
+			if (name !== datetime.name) {
+				updateEntity({ name });
+			}
+		},
+		[datetime.cacheId]
```

Comment:
what about the case when name is changed, but `cacheId` remains the same?
let's add `datetime.name` in deps just for sanity check

Reply:
`cacheId` changes for every change in an entity.